cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE LambdaCase #-}
+
+module Cardano.CLI.EraBased.Commands.Genesis
+  ( GenesisCmds (..)
+  , renderGenesisCmds
+  ) where
+
+import           Cardano.Api.Shelley
+
+import           Cardano.Chain.Common (BlockCount)
+import           Cardano.CLI.Types.Common
+
+import           Data.Text (Text)
+
+data GenesisCmds era
+  = GenesisCreate
+      KeyOutputFormat
```

Comment:
Records types here would be useful, but can be done in a later PR.

Reply:
I agree, 4 `Word`s in a row are a bit confusing.